src/types/resultType.ts

```diff
@@ -0,0 +1,14 @@
+type userResultType = {
+    useranswers:Array<number|null|undefined>;
+    quizes: {answerIndex:number, option:string[], points:number, question:string}[];
+    score:number;
+    isPassed:boolean;
+    quizdetail: {quiz_name:string;
+        quiz_points:number;
+        quiz_question_count:number;
+        quiz_type:string;
```

Comment:
Try using Camel Case. Apart from this everything is good

Reply:
@varun1233 basically these values are coming from DB so I just don't want them to remember every time basically TS will help with intellesense thing